python/opendataloader-pdf-mcp/src/opendataloader_pdf_mcp/server.py

```
@@ -171,6 +174,195 @@ def convert_pdf(
+
+@mcp.tool()
+def get_artifact(job_id: str, source: str = "primary") -> str:
+    """Retrieve converted content for a completed job.
+
+    Args:
+        job_id: The job ID returned by submit_pdf.
+        source: Which artifact to return. Values: primary (default), java, mineru-json.
+    """
+    job = _job_manager.get(job_id)
+    if job.status == JobStatus.FAILED:
+        raise RuntimeError(f"job {job_id} failed: {job.error}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)